src/makeCalculator.js

```diff
@@ -37,7 +37,45 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let sum = 0;
+
+  const calcMethods = {
+    reset() {
+      sum = 0;
+
+      return this;
+    },
+    add(num) {
+      sum += num;
+
+      return this;
+    },
+    subtract(num) {
+      sum -= num;
+
+      return this;
+    },
+    divide(num) {
+      sum /= num;
+
+      return this;
```

Comment:
why do you need this in methods `add`, `subtract`, `divide`, `multiply`, we use those methods with help of `operate` method, so we don't need to `return this` in this case